srml/aura/src/lib.rs

```diff
@@ -75,8 +128,10 @@ impl InherentDataProvider {
 	}
 }
 
+/// Methods to specify the provision of inherent data
```

Comment:
documenting trait should be made on trait itself, not the implementation imo, to avoid duplication